userspace/libsinsp/test/events_fspath.ut.cpp

```diff
@@ -181,42 +185,10 @@ class fspath : public sinsp_with_test_input
 		sinsp_evt* evt = add_event_advance_ts_v(increasing_ts(), 1, event_type, n, args);
 		va_end(args);
 
-		verify_fields(evt, expected_name, expected_name, NULL, NULL, NULL, NULL);
-	}
-
-	void test_exit_path_raw(const char *expected_name,
```

Comment:
We could convert `test_exit_path_raw` to `test_exit_path`. That way the unit tests are clearer and more functional.